### ISSUE TITLE
All files locked for editing

### ISSUE THREAD

**Issue by cramjaco** [astrada/google-drive-ocamlfuse#490]
I am working on ubuntu 18.04 and have successfully mounted a directory to google drive. I find if I try to open a file locally eg by running `libreoffice "Untitled document.odt"` libreoffice gives me the error message

>  Document file "Untitled%40document.odt' is locked for editing by 
> Unknown User
> Open document read-only or open a copy of the document for editing.

I notice that the file permissions of all of the files that are created by google drive in this directory are read only for user group and everyone. I tried giving myself write permissions with `chmod u+w` 'Untitled document.odt' but the permissions appear to remain the same and the problem persists.

Is there a fix or work around for this, or do I just treat this directory as a read only parallel of my google drive files?

**Comment by astrada** [comment]
No, it's by design. The .odt is exported from the original Google's internal format. So if you modify it with a tool that isn't the web editor, there is the risk to mess up layout/style/etc. If you want, you can set `document_format=desktop` (https://github.com/astrada/google-drive-ocamlfuse/wiki/Configuration#document-export-formats) in the configuration, remount with `-cc` to obtain a desktop file that you can double-click to open the web editor.
